Source/UI/NewUI/BillingForm/Factory/BillingFormFactory.swift

```diff
@@ -19,12 +19,12 @@ public struct BillingFormFactory {
     }
 
 
-    static func getBillingFormViewController(billingFormStyle: BillingFormStyle?, delegate: BillingFormViewModelDelegate) -> (BillingFormViewModelDelegate?, UIViewController?) {
+    static func getBillingFormViewController(billingFormStyle: BillingFormStyle?, delegate: BillingFormViewModelDelegate) -> (BillingFormViewModelDelegate?, UINavigationController?) {
         guard let billingFormStyle = billingFormStyle, !billingFormStyle.cells.isEmpty
         else { return (nil, nil) }
-        
         let viewModel = DefaultBillingFormViewModel(style: billingFormStyle, initialCountry: "", delegate: delegate)
-        return (viewModel.delegate, BillingFormViewController(viewModel: viewModel))
+        let navigationController = UINavigationController(rootViewController: BillingFormViewController(viewModel: viewModel))
+        return (viewModel.delegate,navigationController )
```

Comment:
I cannot find the consumer changing to receive a UINavigationController. But why should our BillingForm come with a NavigationController attached?

Also, weird spacing in this tuple return

Reply:
I am not sure [strange](https://www.dictionary.com/browse/weird) is an inappropriate adjective to describe _a lack of space ahead and an extra space following  **navigationController**_.

I appreciate the feedback on my _uncanny_ use of English and I will aim to improve it if you think that is a shortcoming of my reviewing style.

Reply:
I think these PR is open for all developers on Github. let's keep professional.